app/models/company.rb

```diff
@@ -0,0 +1,5 @@
+class Company < ActiveRecord::Base
+  belongs_to :employee
```

Comment:
if company has_many employees, why belong_to employee?

Reply:
For admin id purpose... I don't have any idea about this.. Zain, please review this again...